src/pybind/mgr/ansible/module.py

```diff
@@ -44,18 +45,21 @@
 
 # Manage groups
 URL_MANAGE_GROUP = "api/v1/groups/{group_name}"
+# URLs for Ansible Runner Operations
+URL_GET_HOSTS = "api/v1/hosts"
```

Comment:
move this a but further up. should be near the definition of `URL_GET_HOST_GROUPS`

Reply:
done